src/components/Forms/Radio/style.ts

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components";
+import { BaseRadioProps } from "./index";
+
+type RadioProps = {
+  disabled?: boolean,
+};
+
+export const StyledRadioWrapper = styled.div<RadioProps>`
+  label{
+    cursor: ${(props) => props.disabled ? 'not-allowed' : 'pointer'};
+  }
+
+  .labelRadioSelected {
+    margin-top: 1rem;
+  }
+
+  .bRCIiE:focus + .Radio__IconContainer-sc-crlwn1-1,
+  .egCExO .Radio__IconContainer-sc-crlwn1-1 {
+    border: none;
+    box-shadow: none;
+    background-color: #FF6112;
+  }
+
+  span {
+    font-weight: 400 !important;
+  }
+
+  .pQAxi:hover .Radio__IconContainer-sc-crlwn1-1,
+  .jsYamK:hover .Radio__IconContainer-sc-crlwn1-1,
+  .pQAxi:active .Radio__IconContainer-sc-crlwn1-1 {
+    border: 1px solid #FF6112;
+    box-shadow: 0px 0px 0px 2px rgb(255 231 218);
+  }
```

Comment:
woah ini bahayaaaa

karena styled-components ini bakal di generate sama styled-components dan cryptic, lebih baik jangan kayak gini caa

refer ke:
https://orbit.kiwi/foundation/design-tokens/

dan kalo dia bisa dikasih prop className, baru di override dari situ

ini bisa breaking, kalo misalnya si orbit nge publish versi baru, karena classnamenya bisa berubah

Reply:
Noted thankyou mas mario, so sorry for my bad
changed...